src/sage/manifolds/catalog.py

```diff
@@ -176,48 +177,71 @@ def Kerr(m=1, a=0, coordinates='BL', names=None):
     from sage.functions.trig import cos, sin
     from sage.manifolds.manifold import Manifold
     from sage.misc.functional import sqrt
+
     M = Manifold(4, 'M', structure='Lorentzian')
     if coordinates == "Kerr":
         if names is None:
-            names = (r't:(-oo,+oo)', r'r:(0,+oo)', r'th:(0,pi):\theta',
-                     r'ph:(-pi,pi):periodic:\phi')
+            names = (
+                r't:(-oo,+oo)',
+                r'r:(0,+oo)',
+                r'th:(0,pi):\theta',
+                r'ph:(-pi,pi):periodic:\phi',
+            )
         else:
-            names = (names[0]+r':(-oo,+oo)', names[1]+r':(0,+oo)',
-                     names[2]+r':(0,pi):\theta',
-                     names[3]+r':(-pi,pi):periodic:\phi')
+            names = (
+                names[0] + r':(-oo,+oo)',
+                names[1] + r':(0,+oo)',
+                names[2] + r':(0,pi):\theta',
+                names[3] + r':(-pi,pi):periodic:\phi',
+            )
         C = M.chart(names=names)
         M._first_ngens = C._first_ngens
         g = M.metric('g')
         t, r, th, ph = C[:]
-        rho = sqrt(r**2+a**2*cos(th)**2)
-        g[0, 0], g[1, 1], g[2, 2], g[3, 3] = -(1-2*m*r/rho**2), 1+2*m*r/rho**2,\
-                rho**2, (r**2+a**2+2*a**2*m*r*sin(th)**2/rho**2)*sin(th)**2
-        g[0, 1] = 2*m*r/rho**2
-        g[0, 3] = -2*a*m*r/rho**2*sin(th)**2
-        g[1, 3] = -a*sin(th)**2*(1+2*m*r/rho**2)
+        rho = sqrt(r**2 + a**2 * cos(th) ** 2)
```

Comment:
why does it space out the square for functions? :\

Reply:
Probably a bug...I'll revert it for now.

Reply:
```suggestion
        rho = sqrt(r**2 + a**2 * cos(th)**2)
```